### ISSUE TITLE
Documentation for V4.3 is not viewable

### ISSUE THREAD

**Issue by Beth-Simpson** [containers/podman#20628]
### Issue Description

I have been using the documentation for podman V4.3 - However, this doc seems to have been removed from view. Does anyone know where the doc for 4.3 has gone? I browsed the development blog through this year and there’s nothing about withdrawal of a version there.
It looks like the branch is still there https://github.com/containers/podman/tree/v4.3 so I’m thinking it might be a doc issue?
![Versions](https://github.com/containers/podman/assets/99408252/7a525a6c-8bff-48f7-bec5-943cd95811f6)

### Steps to reproduce the issue

Steps to reproduce the issue
1. Go to https://docs.podman.io/en/latest/Introduction.html, and look for v4.3

### Describe the results you received

The documentation for podman (https://docs.podman.io/en/latest/Introduction.html) version 4.3 seems to have been removed from view. 

### Describe the results you expected

Expected this to be viewable on the documentation.

### podman info output

```yaml
Documentation issue
```

### Podman in a container

No

### Privileged Or Rootless

None

### Upstream Latest Release

Yes

### Additional environment details

Not Applicable

### Additional information

Additional information like issue happens only occasionally or issue happens with a particular architecture or on a particular setting

**Comment by ashley-cui** [comment]
@baude @TomSweeneyRedHat @umohnani8 I think y'all are the admins for our readthedocs. Any idea on why v4.3 doesn't show up? https://readthedocs.org/projects/podman/

**Comment by TomSweeneyRedHat** [comment]
@GabiNecasova FYI

**Comment by TomSweeneyRedHat** [comment]
I'm seeing it on docs.podman.io when I go out there now.  However, the top line of the page seems off: https://docs.podman.io/en/latest/_static/api.html?version=v4.7

Shows "supports a RESTful API for the Libpod library (4.0.0)" rather than 4.7.0 as I would expect.  

Also, the v3.1 docs show (3.2.0) here?

I'm not sure what's going on there.

**Comment by Luap99** [comment]
> I'm seeing it on docs.podman.io when I go out there now. However, the top line of the page seems off: https://docs.podman.io/en/latest/_static/api.html?version=v4.7
> 
> Shows "supports a RESTful API for the Libpod library (4.0.0)" rather than 4.7.0 as I would expect.
> 
> Also, the v3.1 docs show (3.2.0) here?
> 
> I'm not sure what's going on there.

You are looking at the API docs which are different from the readthedocs docs (which this issue is about). 
The API swagger docs are already a lost because nobody ever remembers to bump the API version in the swagger so it will always show you the wrong version.

**Comment by Luap99** [comment]
The normal readthedocs version are all automatically managed by readthedocs as they build based on the git tags and the main branch (latest), all the versions are listed here https://readthedocs.org/projects/podman/versions/. There are more missing between 4.2 and 4.5 but it is not clear why?

I know @baude fixed a build problem with https://github.com/containers/podman/pull/20340 so maybe we just hope it works going forward.

**Comment by Luap99** [comment]
Also maybe change the repo in options form https://github.com/containers/libpod to https://github.com/containers/podman, I doubt that is a problem as github still links the old location but who knows.

**Comment by Beth-Simpson** [comment]
I wanted to ask if there is an outlook as to when this would be resolved?

**Comment by Beth-Simpson** [comment]
@TomSweeneyRedHat Hi Tom, I was wondering whether you had an outlook on when this will be resolved and the V4.3 docs will be viewable again?

**Comment by TomSweeneyRedHat** [comment]
@Beth-Simpson sorry, but I'm clueless on this one.  @baude any thoughts?

**Comment by jt-nti** [comment]
I've had a look and I think every version you want to publish requires the new `.readthedocs.yaml` on the branch. I've created PR #21296 to add one to the v4.3 branch. Without it, I saw a "No default configuration file found at repository's root." error on the version v4.3 build in readthedocs.

The other thing to check is that the v4.3 version is still active:

<img width="513" alt="image" src="https://github.com/containers/podman/assets/3535067/1c81baca-4658-483c-8dd8-11c819af22ce">

With an active version and the `.readthedocs.yaml` file in the v4.3 branch, the docs appeared ok on my experimental project... 

https://podman-jt.readthedocs.io/en/v4.3/
